pkg/state/history_storage.go

```diff
@@ -194,6 +196,16 @@ var properties = map[blockchainEntity]blockchainEntityProperties{
 		needToCut:    true,
 		fixedSize:    false,
 	},
+	blockRewardAtHeight: {
+		needToFilter: true,
+		needToCut:    true,
+		fixedSize:    false,
```

Comment:
We can set `needToCut` to false to store all historical values of `blockRewards` and `totalWavesAmount`. For example look at `featureVote`.